modules/integration/tests-integration/tests-patches/src/test/java/org/wso2/carbon/esb/mediators/clone/ESBJAVA4913HandleExceptionTest.java

```diff
@@ -0,0 +1,72 @@
+package org.wso2.carbon.esb.mediators.clone;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.test.utils.http.client.HttpRequestUtil;
+import org.wso2.carbon.integration.common.admin.client.LogViewerClient;
+import org.wso2.carbon.logging.view.stub.types.carbon.LogEvent;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
+
+import java.io.IOException;
+import java.rmi.RemoteException;
+
+/**
+ * Test clone mediator if it reach error sequence on error.
+ * This test case is for fix done for ESBjAVA-4913
+ */
+public class ESBJAVA4913HandleExceptionTest extends ESBIntegrationTest {
+    private LogViewerClient logViewerClient = null;
+
+    @BeforeClass(alwaysRun = true)
+    public void deployService() throws Exception {
+        // Initializing server configuration
+        super.init();
+        // Deploying the artifact defined in the cloneMediatorHandleException.xml.
+        loadESBConfigurationFromClasspath("/artifacts/ESB/mediatorconfig/clone/cloneMediatorHandleException.xml");
+        logViewerClient = new LogViewerClient(contextUrls.getBackEndUrl(), getSessionCookie());
```

Comment:
call the logViewerClient.clearLogs(). Then it will clear the old logs and only retrieve the logs thereafter when calling logViewerClient.getAllSystemLogs()